src/components/screens/votingQueue/editor/editor.js

```diff
@@ -131,7 +132,10 @@ const Editor = ({
 
   const { added, edited, removed } = useMemo(() => getVoteStats(votes), [votes]);
   const feedback = validateVotes(votes, account.balance, fee.value, t);
-  const isCTADisabled = feedback.error || Object.keys(changedVotes).length === 0;
+  const areVotesInValid = Object.values(votes).some(vote =>
+    (vote.unconfirmed === '' || vote.unconfirmed === undefined));
```

Comment:
I think we should've done this in the validator function, alongside the other checks.

Reply:
I have moved this to the validateVotes function